external/fv3fit/fv3fit/train.py

```diff
@@ -17,32 +14,28 @@
 def get_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "data_path", nargs="*", type=str, help="Location of training data"
+        "train_config", type=str, help="path of training configuration yaml file",
```

Comment:
Nit: the documentation for this CLI ends up being a little sparse (see current: http://vulcanclimatemodeling.com/docs/fv3fit/commandline.html). Providing somewhat more verbose help strings might be worth it, for example specifying what the types of configs are that are now required for training and data.

Reply:
Do you think it would be sufficient to mention the class names in the help strings here, along with the header comment "A command-line interface is available using python -m fv3fit.train. For more detail on the format of the configuration file, see configuration."?

Reply:
That sounds good to me

Reply:
Done.

Reply:
One other docs issue: fv3fit/docs/configuration.rst is stale and should get deleted or updated to autodoc point at the right thing, since it's picked up by sphix.